homeassistant/components/light/decora.py

```diff
@@ -39,14 +39,18 @@ def wrapper_retry(device, *args, **kwds):
         """Try send command and retry on error."""
         # pylint: disable=import-error
         import decora
+        import bluepy
```

Comment:
Oups, I missed this one because decora depends on bluepy... I will add it in the requirements and run the scripts...
Thanks !